samples/net/secure_mqtt_sensor_actuator/README.rst

```diff
@@ -0,0 +1,185 @@
+
+.. list-table::
+
+		* - :file:`prj.conf`
+			- Default config: MQTT Secure, DHCP
+
+		* - :file:`overlay_static_insecure.conf`
+			- Simplified config for quick testing: Unsecure MQTT, static IP address
+
+**Default Config:**
+
+Using the default config, the application will use DHCP to acquire an IP address and attempt
+to securely connect to an MQTT broker using TLS 1.2.
+
+- The MQTT broker to which the board will connect is specified by :kconfig:option:`CONFIG_MQTT_BROKER_IPV4_ADDR`
+  and :kconfig:option:`CONFIG_MQTT_BROKER_HOSTNAME`. By default, these are set up to connect to test.mosquitto.org.
+  Please verify that the IP address specified for test.mosquitto.org is correct as it may change over time.
+- Connecting securely using TLS, requires the inclusion of the broker's CA certificate in the application.
+- Download the CA certificate in DER or PEM format from https://test.mosquitto.org
+- In :file:`tls_config/cert.h`, set ``ca_certificate[]`` to the contents of the cert.
+- By connecting the board to your internet router, it should be automatically be assigned an IPv4 address using DHCP.
+- The application will then attempt to connect to the public Mosquitto broker and begin publishing data.
+- It is also possible to connect securely to a locally hosted MQTT broker. This will require provisioning of
+  certificates. The CA cert should be included in the build as described above.
+  The broker IP address and hostname Kconfig options should be configured to suit the local broker settings.
+  See https://mosquitto.org/man/mosquitto-tls-7.html for more info on setting up TLS support for Mosquitto locally.
+- A DHCP server can be installed on the host PC to handle assigning an IP to the board e.g. dnsmasq (Linux) or DHCP Server for Windows (Windows).
+- Build the sample with default config:
+
+.. zephyr-app-commands::
+	 :zephyr-app: samples/net/secure_mqtt_sensor_actuator
+	 :board: adi_eval_adin1110ebz
+	 :goals: build
+	 :compact:
+
+**Quick Test Config:**
+
+Use the overlay_static_insecure.conf Kconfig overlay to disable TLS and DHCP.
+This config requires connecting to a locally hosted Mosquitto MQTT broker.
+
+- In :file:`overlay_static_insecure.conf`, set the IP address of the board and the Mosquitto
+  broker (i.e. IP address of Ethernet port on host PC). These addresses must be in the
+  same subnet e.g. 192.0.2.1 and 192.0.2.2.
+- On your host PC, install Mosquitto.
+- Create a file called ``unsecure.conf`` with the following content:
+
+.. code-block:: console
+
+		listener 1883 0.0.0.0
+		allow_anonymous true
+
+
+- Start a Mosquitto broker using the configuration file:
+
+.. code-block:: console
+
+		$ sudo mosquitto -v -c unsecure.conf
+
+- Build the sample with quick test config:
+
+.. zephyr-app-commands::
+	 :zephyr-app: samples/net/secure_mqtt_sensor_actuator
+	 :board: adi_eval_adin1110ebz
+	 :conf: overlay_static_insecure.conf
```

Comment:
This command is wrong, provides an overlay file as a config file (I. e. `-DCONF_FILE` instead of `-DOVELRAY_CONFIG`).

Reply:
I think I have fixed this - can you check if it is correct now?

Reply:
Looks ok now